.pipelines/e2e-with-billing.yml

```diff
@@ -16,6 +16,7 @@ stages:
         location: $(LOCATION)
         subscription: $(e2e-subscription)
         azureDevOpsE2EJSONSPN: $(aro-v4-e2e-devops-spn)
+        aroVersionStorageAccount: $(aro-version-storage-account)
```

Comment:
should this also say `rpMode: int` here?

Reply:
Production?